src/Cell.cpp

```diff
@@ -583,8 +619,9 @@ CellBasic* CellBasic::clone() {
 
 /**
  * @brief Subdivides the Cell into clones for fuel pin angular sectors.
+ * @param subcells an empty vector to store all subcells
  */
-void CellBasic::sectorize() {
+void Cell::sectorize(std::vector<Cell*>* subcells) {
```

Comment:
We don't usually pass pointers to vectors, just the vectors themselves.  What's the rationale here?